src/api/app/helpers/webui/maintenance_incident_helper.rb

```diff
@@ -58,13 +58,15 @@ def category_cell(incident, patchinfo)
     end
   end
 
-  def packages_cell(incident, release_targets_ng)
-    release_target = release_targets_ng.values.first
-    return if release_target[:packages].blank?
-    first_pkg = release_target[:packages].first
+  def packages_cell(incident)
+    packages = incident.packages.joins(:flags).where(flags: { flag: :build, status: 'enable', repo: incident.release_targets.
```

Comment:
this is a little a lot of AR for a helper IMO. Perhaps move this into a function of its own, either directly in the incident or a service class for incidents.